lib/lcdbwf/R/functional_enrichment.R

```diff
@@ -101,11 +103,14 @@ collapse_names <- function(res_list, config, sep='*'){
 #'   lfc_thresh from the config.
 #' @param kind One of "OR" for overrepresentation or "GSEA" for gene set
 #'   enrichment analysis.
+#' @param universe background genes. If missing, all genes listed in the database 
+#'   (e.g. TERM2GENE table) will be used as background. See more info in
+#'   https://github.com/YuLab-SMU/clusterProfiler/blob/devel/R/enricher.R#L8
```

Comment:
This line may change over time as clusterProfiler develops. That said, this line 8 says the same thing this help string says, so I don't think the link is needed.

Reply:
ok, I'm deleting the url.